StRoot/StPicoEvent/StPicoEvent.cxx

```diff
@@ -27,6 +27,12 @@ StPicoEvent::StPicoEvent(): TObject(),
   mRefMultHalfNegEast(0), mRefMultHalfPosEast(0), mRefMultHalfNegWest(0), mRefMultHalfPosWest(0),
   mGRefMult(0), mNumberOfGlobalTracks(0), mbTofTrayMultiplicity(0), mNHitsHFT{},
   mNVpdHitsEast(0), mNVpdHitsWest(0), mNTofT0(0), mVzVpd(-999.),
+
+  // Start: bassam
+  // In the following, the factors of 1000. serve to convert times back and forth between external nsec and internal psec
+  mNTofT0Can(0), mTStart(-9999*1000), mTCanFirst(99999*1000), mTCanLast(-99999*1000), mNVpdEGoodHits(0), mNVpdWGoodHits(0), mEarliestVpdEHit(99999*1000), mEarliestVpdWHit(99999*1000), mClosestVpdEHit(99999*1000), mClosestVpdWHit(99999*1000), mLatestVpdEHit(-99999*1000), mLatestVpdWHit(-99999*1000),
+  // End: bassam
+
```

Comment:
Please remove empty lines and the lines that contain // Start .... // End:.
Also consider the latter (start-end thing)  for the other pieces of code.

Reply:
Thank you for the feedback! I addressed this in my new commit.